atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/TransactionManager.java

```diff
@@ -316,6 +316,7 @@ default boolean isInitialized() {
     TimelockService getTimelockService();
 
     // todo(gmaretic): implement
+    @DoNotDelegate
```

Comment:
technically, this is fine right? but for consistency sake/future proofing I'm guessing?